src/services/ticker.js

```diff
@@ -11,30 +11,43 @@ type TickerAPIParams = {|
   +source: 'coinmarketcap',
 |}
 
-const { tickerAPIOptions } = config
-
+const { tickerAPIOptions }: AppConfig = config
+const AVAILABLE_CURRENCIES: string[] = Object.keys(CURRENCIES)
 const TICKER_API: string = getENVVar('_TICKER_API__') || __DEFAULT_TICKER_API__
 
-function callApi(params: TickerAPIParams, retryCount: number = 4): Promise<any> {
-  const requestInfo: RequestInfo = `${TICKER_API}/v2/quotes`
+function handleRequestError(
+  params: TickerAPIParams,
+  retryCount: number,
+) {
+  if (retryCount > 0) {
+    // eslint-disable-next-line no-use-before-define
+    return callApi(params, (retryCount - 1))
```

Comment:
Looks like we need `setTimeout` here

Reply:
I remember, we have separate [task](https://jibrelnetwork.atlassian.net/browse/JWWEB-987) for it.